package.json

```diff
@@ -170,6 +171,7 @@
     "jest-junit": "^16.0.0",
     "jsonwebtoken": "^9.0.2",
     "lint-staged": "^15.2.7",
+    "lodash": "^4.17.21",
```

Comment:
I've added lodash / underscore to support a test I wanted to fix whilst I was here (the next file in the diff 👇 )
I'm not entirely happy with introducing a dependency , but at least it's a big/well maintained and supported one 🤷‍♂️